lang/CMakeLists.txt

```diff
@@ -182,6 +182,10 @@ if(SC_QT OR SC_IDE)
 	message(STATUS "Found Qt: " ${QtCore_location} )
     get_filename_component(QT_BIN_PATH ${QtCore_location} DIRECTORY CACHE)
 
+  if(SC_USE_QTWEBENGINE)
+    message(STATUS "sclang: Building with QtWebEngine")
+    target_compile_definitions(libsclang PUBLIC SC_USE_QTWEBENGINE)
+  endif()
```

Comment:
since the surrounding code mixes tabs and 4-space indentation, i don't think there's really any reasonable whitespace decision i can make here that doesn't involve reformatting the entire file.

Reply:
4 spaces will prevent a diff here in the future, please do that

Reply:
fair enough